include/render/gles2.h

```diff
@@ -97,6 +97,16 @@ const enum wl_shm_format *get_gles2_wl_formats(size_t *len);
 
 struct wlr_gles2_texture *gles2_get_texture(
 	struct wlr_texture *wlr_texture);
+struct wlr_gles2_renderer *gles2_get_renderer(
+		struct wlr_renderer *wlr_renderer);
+
+struct wlr_texture *wlr_gles2_texture_from_pixels(struct wlr_renderer *renderer,
```

Comment:
Private functions need to have their `wlr_` prefix stripped, otherwise they will be visible in the resulting dynamic library.

Reply:
done.